### PR TITLE
fix broken sizing of floating widgets 

### PR DESCRIPTION
**Pull request by tmcgee**

### CODE REVIEW

**Comment by tmcgee** [comment]
@roemhildtg bump

File: viewer/js/viewer/_WidgetsMixin.js [hunk 1]
```diff
@@ -297,7 +297,7 @@ define([
             widgetConfig.paneOptions = lang.mixin({
                 id: parentId,
                 title: widgetConfig.title,
-                class: parentId,
+                class: parentId + ' ' + widgetConfig.type + 'Widget',
```
Comment (green3g) [inline viewer/js/viewer/_WidgetsMixin.js:300]: Should this  be surrounded in quotes: `'class': parentId...`?

Comment (tmcgee) [reply viewer/js/viewer/_WidgetsMixin.js:300]: Obviously that can't hurt though it wasn't quoted in the original commit.

Comment (green3g) [reply viewer/js/viewer/_WidgetsMixin.js:300]: Right, I just noticed the file above had quoted it, and this one did not. I don't remember is it IE that has issues with this?

Comment (tmcgee) [reply viewer/js/viewer/_WidgetsMixin.js:300]: pretty sure that's not an issue any more with the modern versions of IE supported by dojo/esri/cmv. The original file dates back to a bygone era with support for IE 8.

Comment (green3g) [reply viewer/js/viewer/_WidgetsMixin.js:300]: Okay good to know.

**Pull request merged**